spec/controllers/checkout_controller_spec.rb

```diff
@@ -315,11 +315,11 @@
   describe '#expire_current_order' do
     it 'empties the order_id of the session' do
       expect(session).to receive(:[]=).with(:order_id, nil)
-      controller.expire_current_order
+      controller.send(:expire_current_order)
```

Comment:
That's a code smell but okay for refactoring purposes. No need to invest the work to rewrite specs now, too.

Reply:
I'll add a bit of explanation for my reasoning here.

A bit of background to the related code changes; previously this method was duplicated in both the checkout controller and the paypal controller. It's needed in both. It was also defined as a public method in both controllers, and clearly shouldn't be a public method.

So what happened in the course of this PR is; this method has moved to a shared Concern that's included in both controllers so it can be defined in just one place instead of being duplicated, and it's also been changed to a private method.

Generally speaking it's not common practise to test private methods, but I see that more as a useful guideline than a hard rule. In this specific case I think there's value in keeping the test coverage on this method regardless of whether it's public or private.